### PR TITLE
Update version links and documentation to reflect v2.0-RC

### PR DESCRIPTION
**Pull request by barbeau**
* In README, for v1.0 and v1.1 RC point to the Git tags for gbfs.md files instead of commit hashs to improve readability. For v2.0 RC, the current link points to open pull requests with the v2.0 tag, of which there aren't any (because they have been merged into the master branch). Instead, point to the current master branch gbfs.md (see below).
* In gbfs.md, change text to reference version to 2.0 RC (all v2.0 candidate changes have already been merged into the master branch).

### CODE REVIEW

**Comment by barbeau** [comment]
After @heidiguenin and I reviewed the master branch and my initial changes in this pull request to update version links in the README, we concluded that for the master branch to accurately reflect the state of the v2.0-RC candidate spec the gbfs.md text needed to be updated. 

To be clear, all of the proposals for v2.0-RC were already voted on and merged, but when they were merged the v2.0-RC text was appended to the existing v1.0/v1.1 text, which made it harder to read.

So, I've made the following changes in this PR (via https://github.com/NABSA/gbfs/pull/221/commits/efaf21d42368774666a558ba593e05c6c08e70d1) so that the master branch reflects the current state of the spec as of v2.0-RC:
* For all new fields or files that were added after v1.0, add the label `added in vX.X` after the field name to indicate in which version they were added
* For all requirement or description changes that changed after v1.0, add the label `as of vX.X` in-line to indicate in which version they changed
* Change all Boolean examples to be true and false (per v2.0-RC)
* Update the ID field type definition to say that bike_id is not persistent so it matches the v2.0-RC description in `free_bike_status.json` (per v2.0-RC)
* Update all example JSON versions to be 2.0
* Fix the version number in the gbfs_versions.json example

@heidiguenin This should be ready for your review! Thanks!

File: gbfs.md [hunk 1]
```diff
@@ -241,19 +241,19 @@ Field Name | Required | Type | Defines
...
+`license_url` | Conditionally required <br/>*(as of v2.0-RC)* | URL | Required if the dataset is  provided under a customized license. A fully qualified URL of a page that defines the license terms for the GBFS data for this system. Do not specify a `license_url` if `license_id` is specified. If the license_id or license_url fields are blank or omitted, this indicates that the feed is provided under the [Creative Commons Universal Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/legalcode). *(as of v2.0-RC)*
+`attribution_organization_name` <br/>*(added in v2.0-RC)* | Optional | If the feed license requires attribution, name of the organization to which attribution should be provided.
```
Comment (heidiguenin) [inline gbfs.md]:
```suggestion
`attribution_organization_name` <br/>*(added in v2.0-RC)* | Optional | String | If the feed license requires attribution, name of the organization to which attribution should be provided.
```

File: gbfs.md [hunk 2]
```diff
@@ -241,19 +241,19 @@ Field Name | Required | Type | Defines
...
+`attribution_url` <br/>*(added in v2.0-RC)* | Optional | URL of the organization to which attribution should be provided.
```
Comment (heidiguenin) [inline gbfs.md]:
```suggestion
`attribution_url` <br/>*(added in v2.0-RC)* | Optional | URL | URL of the organization to which attribution should be provided.
```

File: gbfs.md [hunk 3]
```diff
@@ -285,12 +285,12 @@ Field Name | Required | Type | Defines
...
-\-&nbsp;`num_docks_available` | Yes<br/>*(beta v2.0-RC)* Conditionally required | Non-negative integer | *Current version:* Number of docks accepting bike returns. <br/>*Beta v2.0-RC:* Required except for stations that have unlimited docking capacity (e.g. virtual stations). Number of functional docks physically at the station. To know if the docks are accepting bike returns, see `is_returning`.
+\-&nbsp;`num_bikes_disabled` | Optional | Non-negative integer | Number of disabled bikes at the station. Vendors who do not want to publicize the number of disabled bikes or docks in their system can opt to omit station capacity (in station_information), `num_bikes_disabled` and `num_docks_disabled` *(as of v2.0-RC)*. If station capacity is published then broken docks/bikes can be inferred (though not specifically whether the decreased capacity is a broken bike or dock).
```
Comment (heidiguenin) [inline gbfs.md]:
```suggestion
\-&nbsp;`num_bikes_disabled` | Optional | Non-negative integer | Number of disabled bikes at the station. Vendors who do not want to publicize the number of disabled bikes or docks in their system can opt to omit station capacity (in station_information), `num_bikes_disabled` and `num_docks_disabled` *(as of v2.0-RC)*. If station capacity is published, then broken docks/bikes can be inferred (though not specifically whether the decreased capacity is a broken bike or dock).
```

File: gbfs.md [hunk 4]
```diff
@@ -299,15 +299,15 @@ Describes bikes that are not at a station and are not currently in the middle of
 Field Name | Required | Type | Defines
 --|--|--|--
 `bikes` | Yes | Array | Array that contains one object per bike that is currently stopped as defined below.
-\-&nbsp;`bike_id` | Yes | ID | Identifier of a bike. *Beta (v2.0-RC):* Identifier of a bike, rotated to a random string, at minimum, after each trip to protect privacy. Note: Persistent bike_id, published publicly, could pose a threat to individual traveler privacy.
+\-&nbsp;`bike_id` | Yes | ID | Identifier of a bike, rotated to a random string, at minimum, after each trip to protect privacy *(as of v2.0-RC)*. Note: Persistent bike_id, published publicly, could pose a threat to individual traveler privacy.
```
Comment (heidiguenin) [inline gbfs.md:303]: This table still seems to have a problem with wrapping.

File: gbfs.md [hunk 5]
```diff
@@ -299,15 +299,15 @@ Describes bikes that are not at a station and are not currently in the middle of
...
 \-&nbsp;`lat` | Yes | Latitude | Latitude of the bike.
 \-&nbsp;`lon` | Yes | Longitude | Longitude of the bike.
 \-&nbsp;`is_reserved` | Yes | Boolean | Is the bike currently reserved? <br /><br /> `1` - Bike is currently reserved. <br /> `0` - Bike is not currently reserved.
```
Comment (heidiguenin) [inline gbfs.md]:
```suggestion
\-&nbsp;`is_reserved` | Yes | Boolean | Is the bike currently reserved? <br /><br /> `true` - Bike is currently reserved. <br /> `false` - Bike is not currently reserved.
```

File: gbfs.md [hunk 6]
```diff
@@ -299,15 +299,15 @@ Describes bikes that are not at a station and are not currently in the middle of
...
 \-&nbsp;`is_disabled` | Yes | Boolean | Is the bike currently disabled (broken)? <br /><br /> `1` - Bike is currently disabled. <br /> `0` - Bike is not currently disabled.
```
Comment (heidiguenin) [inline gbfs.md]:
```suggestion
\-&nbsp;`is_disabled` | Yes | Boolean | Is the bike currently disabled (broken)? <br /><br /> `true` - Bike is currently disabled. <br /> `false` - Bike is not currently disabled.
```

**Review by heidiguenin — COMMENTED** [review]
There were a couple boolean changes I didn't catch right away plus some other small revisions.

File: gbfs.md (hunk 1 above)
Comment (barbeau) [reply gbfs.md]: Thanks! I fixed in https://github.com/NABSA/gbfs/pull/221/commits/8d7c56ef46905d7594783ea70e888b45f9f1a244.

File: gbfs.md (hunk 2 above)
Comment (barbeau) [reply gbfs.md]: Thanks! I fixed in https://github.com/NABSA/gbfs/pull/221/commits/8d7c56ef46905d7594783ea70e888b45f9f1a244.

File: gbfs.md (hunk 4 above)
Comment (barbeau) [reply gbfs.md:303]: Yes, I saw that too. But I'm not sure how to fix it. It seems to be formatted the same as station_information.json, and that wraps...

**Comment by barbeau** [comment]
@heidiguenin I just realized that we are missing a type definition for URIs at the top of the document too - I just added that in https://github.com/NABSA/gbfs/pull/221/commits/b5e0d8268a41b58946f0506262baee0f58afe4a2.

**Comment by barbeau** [comment]
@heidiguenin While was trying to figure out why the table `free_bike_status.json` isn't wrapping, I saw that [GitHub's official markdown table documentation](https://help.github.com/en/github/writing-on-github/organizing-information-with-tables) says:

>There must be at least three hyphens in each column of the header row.

We've only had two hyphens `--` in each column of the header rows. I changed to use three hyphens `---` in https://github.com/NABSA/gbfs/pull/221/commits/b7e14739cdf5e57d8bb6af755d6dfd2e1d8c0343. This unfortunately didn't fix  `free_bike_status.json` wrapping, but it did make it much easier to see markdown rendering previews with editors like IntelliJ.

The top image is the before (which doesn't render the table in IntelliJ), while the bottom image is after (which does render the table in IntelliJ). It also makes the in-line cell separators easier to see in the raw text view.

image

**Comment by heidiguenin** [comment]
@barbeau thanks for catching that! I'd suggest we merge these changes and keep sleuthing on why `free_bike_status.json` doesn't wrap.

**Review by heidiguenin — APPROVED** [review]
@barbeau  Looking good now! I'd suggest we merge these changes and keep sleuthing on why free_bike_status.json doesn't wrap.

**Comment by barbeau** [comment]
@heidiguenin Sounds good to me - I was just looking at the GitHub flavored markdown spec for tables (https://github.github.com/gfm/#tables-extension-) but I don't see an obvious answer there either.

**Pull request merged**